src/components/Header/index.js

```diff
@@ -1,97 +1,88 @@
-import React, { useState } from 'react'
+import React, { Fragment, useState } from 'react'
 
 import styled from 'styled-components'
 import Flex from 'styled-flex-component'
 import { Padding } from 'styled-components-spacing'
+import breakpoint from 'styled-components-breakpoint'
+import remcalc from 'remcalc'
 
+import navLinks from './navLinks'
 import { Row, Col, Grid } from '../grid'
 import Hamburger from './Hamburger'
 import Overlay from './Overlay'
 import TopNav from './TopNav'
 import SideNav from './SideNav'
 
-const FixedHeightFlex = styled(Flex)`
-  height: 84px;
+const StyledPadding = styled(Padding)`
+  display: flex;
+  height: ${remcalc(120)};
+  align-items: center;
 `
 
-const navLinks = [
-  {
-    label: 'Services',
-    dropdownItems: [
-      {
-        label: 'Engineering',
-        to: '/engineering/'
-      },
-      {
-        label: 'Design',
-        to: '/design/'
-      },
-      {
-        label: 'Training',
-        to: '/training/'
-      }
-    ]
-  },
-  {
-    label: 'Our work',
-    to: '/our-work/'
-  },
-  {
-    label: 'Blog',
-    href: 'https://medium.com/yld-engineering-blog/'
-  },
-  {
-    label: 'About',
-    dropdownItems: [
-      {
-        label: 'Our team',
-        to: '/about-us/'
-      },
-      {
-        label: 'Contact',
-        to: '/contact/'
-      }
-    ]
-  },
-  {
-    label: 'Join us',
-    to: '/join-us/'
-  }
-]
+const StyledGrid = styled(Grid)`
+  position: fixed;
+  height: ${remcalc(84)};
+  background: ${props =>
+    props.theme.colors[props.isSpecialityPage ? 'blueBg' : 'white']};
+  width: 100%;
+  max-width: unset;
+  z-index: 100;
+  box-shadow: ${props => `0 9px 9px -9px ${props.theme.colors.border}`};
```

Comment:
This has been updated since your comment ;) Only appears on scroll now. Also not a box-shadow anymore as needed linear-gradient instead to achieve the desired effect.